elliottlib/ccs_image_list.py

```diff
@@ -0,0 +1,45 @@
+import xmlrpclib
+import re
+
+from elliottlib import constants
+
+
+def generate(image_advisory_id):
+    """Generate an image list of a given advisory in the format we usually send
+    to CCS (docs team)
+
+    :param str image_advisory_id: ID of the main image advisory
+
+    :return: str with a list of images from the advisory in a format CCS consumes
+    """
+    errata_tool = xmlrpclib.ServerProxy(constants.errata_xmlrpc_url)
```

Comment:
Created `ErrataXMLRPC` object outside so it can be imported anywhere by `elliottlib.errata.ErrataXMLRPC`: https://github.com/openshift/elliott/pull/90/files#diff-039448b14756b1d04ef08dbd7cd63fb8R29